### PR TITLE
Review Sign-in content

### PR DESCRIPTION
**Pull request by anikbrazeau**
Minor tweaks to content in the screens when signing in, including:

- Changing awkward phrasing : "Not received email?" which might confuse users.
- Removing redundancy : "to sign in to GC Notify" which users already know at this point.

### CODE REVIEW

File: app/templates/views/two-factor-email.html [hunk 1]
```diff
@@ -14,14 +14,14 @@
   <div class="md:w-2/3 float-left py-0 px-0 px-gutterHalf box-border">
     <h1 class="heading-large">{{ _(title) }}</h1>
     {% set continue = _('Continue') %}
-    {% set link_txt = _('Not received an email?') %}
+    {% set link_txt = _('Didn't get an email?') %}
```
Comment (anikbrazeau) [inline app/templates/views/two-factor-email.html:17]:
```suggestion
    {% set link_txt = _("Didn't get an email?") %}
```

File: app/templates/views/two-factor-email.html [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
     {% if requires_email_login %}
       <p>
         {{ _("For added security, GC Notify has sent you an email message with a security code to confirm you still control a valid Government email address.") }}
       </p>
     {% else %}
       <p>
-        {{ _('We’ve sent you an email with a security code to sign in to GC Notify.') }}
+        {{ _('We’ve sent you an email with a security code.') }}
```
Comment (anikbrazeau) [inline app/templates/views/two-factor-email.html:24]:
```suggestion
        {{ _("We’ve sent you an email with a security code.") }}
```

File: app/templates/views/two-factor-sms.html [hunk 3]
```diff
@@ -14,7 +14,7 @@
   <div class="md:w-2/3 float-left py-0 px-0 px-gutterHalf box-border">
     <h1 class="heading-large">{{ _('Check your phone messages') }}</h1>
     {% set continue = _('Continue') %}
-    {% set link_txt = _('Not received a text message?') %}
+    {% set link_txt = _('Didn't get a text message?') %}
```
Comment (anikbrazeau) [inline app/templates/views/two-factor-sms.html:17]:
```suggestion
    {% set link_txt = _("Didn't get a text message?") %}
```

File: tests/app/main/views/test_two_factor.py [hunk 4]
```diff
@@ -64,7 +64,7 @@ def test_should_render_email_two_factor_page(
     response = client.get(url_for("main.two_factor_email_sent"))
     assert response.status_code == 200
     page = BeautifulSoup(response.data.decode("utf-8"), "html.parser")
-    assert page.select_one("main p").text.strip() == ("We’ve sent you an email with a security code to sign in to GC Notify.")
+    assert page.select_one("main p").text.strip() == ("We’ve sent you an email with a security code.")
```
Comment (jimleroyer) [inline tests/app/main/views/test_two_factor.py:67]: Fixing tests now?! 😮 👍

**Review by jimleroyer — APPROVED** [review]
:shipit:

File: tests/app/main/views/test_two_factor.py (hunk 4 above)
Comment (anikbrazeau) [reply tests/app/main/views/test_two_factor.py:67]: With big thanks to @sastels :)

**Pull request merged**